<launch>
    <!-- 设置参数 -->
    <arg name="rtsp_url" default="rtsp://192.168.144.25:8554/main.264" />
    <arg name="camera_info_path" default="$(find sunray_tutorial)/config/head_camera.yaml" />
    
    <!-- RTSP拉流图像发布节点 -->
    <node pkg="sunray_gimbal" type="rtsp_image_publisher" name="rtsp_image_publisher" output="screen">
        <!-- RTSP流地址，可根据实际修改 -->
        <param name="rtsp_url" value="$(arg rtsp_url)" />
        <!-- 相机内参yaml路径 -->
        <param name="camera_info_path" value="$(arg camera_info_path)" />
    </node>
</launch>
